scripts/discourse-sync/client.py

```diff
@@ -0,0 +1,22 @@
+import pydiscourse
```

Comment:
Not sure if the Juju team have any style guides on this, but usually you'd put a docstring at the top of the module that briefly explains what it's for.